src/benchmarks/micro/libraries/System.Numerics.BitOperations/Perf_BitOperations.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Numerics;
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+using System.Runtime.CompilerServices;
+
+namespace System.Numerics.Tests
+{
+    [BenchmarkCategory(Categories.Libraries, Categories.SIMD)]
+    public class Perf_BitOperations
+    {
+        private static Random s_random = new Random(5);
+        private static uint[] input_uint = GenerateRandomValues<uint>();
```

Comment:
We have a dedicated type that allows for generating [always the same input data](https://github.com/dotnet/performance/blob/master/docs/microbenchmark-design-guidelines.md#Always-the-same-input-data):

https://github.com/dotnet/performance/blob/dd9884c879def10a8a96dd3cefc5d6c080c32253/src/harness/BenchmarkDotNet.Extensions/ValuesGenerator.cs#L44

Could you please use it (and extend with new method if needed)?